src/Microsoft.SymbolStore/KeyGenerators/ELFFileKeyGenerator.cs

```diff
@@ -62,7 +62,7 @@ public override IEnumerable<SymbolStoreKey> GetKeys(KeyTypeFlags flags)
                     bool symbolFile = false;
                     try
                     {
-                        symbolFile = Array.Exists(_elfFile.Sections, section => ((section.Name == ".debug_info") || (section.Name == ".zdebug_info")));
+                        symbolFile = Array.Exists(_elfFile.Sections, section => (section.Name.StartsWith(".debug_") || section.Name.StartsWith(".zdebug_")));
```

Comment:
Not sure how important this is but there are a lot of other sections that begin with .debug_/.zdebug_ that I assume are only in a binary containing symbols but wouldn't doing `.StartsWith(".debug_info")/.StartsWith(".zdebug_info")` be more precise and still cover this case?

Reply:
Yep, I agree, we can err on the side of caution.